src/joystream-lib/types.ts

```diff
@@ -42,4 +54,9 @@ export type ExtrinsicStatusCallbackFn = (status: ExtrinsicStatus.Unsigned | Extr
 export type ExtrinsicResult<T> = {
   block: number
   data: T
+  contentIds?: ExtrinsicContentIds
+}
+
+export type InputAssets = {
```

Comment:
Let's improve this type, similar to what we did with `ExtrinsicContentIds`, to not allow arbitrary fields, but rather have options predefined